django/core/management/commands/shell.py

```diff
@@ -21,6 +21,14 @@ class Command(BaseCommand):
 
     requires_system_checks = []
     shells = ["ipython", "bpython", "python"]
+    default_imports = [
```

Comment:
I think this has been defined as a class attribute to allow folks to override it and change the "default utilities" that Django imports by default. 
We already have some docs which suggest folks should just override `get_auto_imports()`. If we think they could/should override `default_imports` we should document it. Otherwise, I would move this within the `get_auto_imports()` method